### PR TITLE
fix layout scrolling

### PR DESCRIPTION
**Pull request by ryanatkn**
Fixes the main layout scrollbars when the space is taller than the viewport (happens on the `Home` view on smaller screens) along with some other minor issues, like using `403` instead of `401`.

### CODE REVIEW

File: src/lib/ui/view/Home.svelte [hunk 1]
```diff
@@ -24,7 +25,7 @@
 	$: communityPersonas = $personasByCommunityId.get($community.community_id)!;
 </script>
 
-<div class="markup">
+<div class="home">
```
Comment (ryanatkn) [inline src/lib/ui/view/Home.svelte:28]: This gets rid of the bullet points.

File: src/lib/ui/WorkspaceHeader.svelte [hunk 2]
```diff
@@ -7,14 +7,18 @@
 	import {format} from 'date-fns';
 
 	const {
-		ui: {expandMainNav},
+		ui: {expandMainNav, expandMarquee},
```
Comment (ryanatkn) [inline src/lib/ui/WorkspaceHeader.svelte:10]: The code here copies the same pattern we use for the luggage button but for the marquee on the right side. Without it the marquee button overlaps the timestamp when it's closed.

File: src/lib/ui/Workspace.svelte [hunk 3]
```diff
@@ -36,15 +36,17 @@
...
+	<div class="space column">
 		<!-- TODO pass stores here instead of dereferncing -->
 		<WorkspaceHeader space={selectedSpace} community={selectedCommunity} />
-		{#if selectedPersona && selectedCommunity && selectedSpace}
-			<SpaceView persona={selectedPersona} community={selectedCommunity} space={selectedSpace} />
-		{:else if selectedPersona && selectedCommunity}
-			<!-- TODO this should be the form, not dialog trigger -- fix after refactoring dialogs -->
-			<SpaceInput persona={selectedPersona} community={selectedCommunity} />
-		{/if}
+		<div class="content">
```
Comment (ryanatkn) [inline src/lib/ui/Workspace.svelte:42]: This `.content` wrapper element now sets the bounds for the loaded space, controlling its scrolling.

Comment (greatbacon) [reply src/lib/ui/Workspace.svelte:42]: Content makes sense to me.

**Pull request merged**